dev_guide/copy_files_to_container.adoc

```diff
@@ -0,0 +1,54 @@
+= Copying local files to a container
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+You can use the CLI to copy local files to a remote directory in a container. 
+
+== Basic Usage
+Support for copying local files to a container is built into
+link:../cli_reference/overview.html[the CLI]:
+
+----
+$ oc rsync <local_dir> <pod>:<remote_dir> [-c <container>] 
+----
+
+For example:
+
+====
+
+----
+$ oc rsync /home/user/source devpod1234:/src
+----
+====
+
+== Requirements
+
+The `oc rsync` command uses the local `rsync` command if present on the client's machine. This requires
+that the remote container also have the `rsync` command.  If `rsync` is not found locally or in the remote 
+container, then a tar archive will be created locally and sent to the container where `tar` will be used to
+extract the files. If `tar` is not available in the remote container, then the copy will fail.
```

Comment:
If rsync is available locally and not in the container, we will fall back to tar. That's what I thought I said up there, but maybe should make it more explicit?